packages/plugin-ext-vscode/src/node/plugin-vscode-deployer-participant.ts

```diff
@@ -25,8 +27,21 @@ export class PluginVSCodeDeployerParticipant implements PluginDeployerParticipan
     protected readonly environments: PluginVSCodeEnvironment;
 
     async onWillStart(context: PluginDeployerStartContext): Promise<void> {
-        const extensionsDirUri = await this.environments.getExtensionsDirUri();
-        context.userEntries.push(extensionsDirUri.withScheme('local-dir').toString());
+        const extensionDeploymentDirUri = await this.environments.getDeploymentDirUri();
+        context.userEntries.push(extensionDeploymentDirUri.withScheme('local-dir').toString());
```

Comment:
Is this releated to the `--plugins=local-dir:../../plugins` cli parameter? Since we separated "drop-in" directories from "deployment" directories, what does the cli-parameter now mean?

Reply:
I don't think this part of the code ever cared about `--plugins`:
https://github.com/eclipse-theia/theia/blob/2d30b295c1158f401d63591eb67b46e93cd6f318/packages/plugin-ext-vscode/src/common/plugin-vscode-environment.ts#L31
Maybe this is a difference between theia plugins and vscode extensions? For vscode extensions, it looks already hard-coded to me.